data/vifm.appdata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-	<id>info.vifm</id>
+	<id>info.vifm.vifm</id>
```

Comment:
`info.vifm.app` would also be a valid id, right?  `info.vifm.vifm` is too weird (`info.vifm` should actually be enough, but looks like spec authors didn't take this common case into account).

Reply:
It would, however "`info.vifm.`" should be the developer/domain name while the last part (`vifm`) should be the app name. On the other hand, there are some apps, for example org.telegram.desktop, that use the same format as you suggested.

Reply:
`org.7-zip.7-zip` is used as an example, so I guess ugly IDs are the norm there.

Thanks for the fixes!